.travis.yml

```diff
@@ -1,16 +1,17 @@
 language: php
 
+dist: trusty
```

Comment:
Because Travis is broken, see for example https://travis-ci.org/wmde/Serialization/jobs/232359747, and https://github.com/travis-ci/travis-ci/issues/7712.